docs/source/contributing.rst

```diff
@@ -0,0 +1,80 @@
+Contributing
+============
+
+Thanks for helping to build ``dask-ml``!
+
+Cloning the Rep
+~~~~~~~~~~~~~~~
+
+Make a fork of the repo and clone that
+
+.. code-block:: none
+
+   git clone https://github.com/<your-github-username>/dask-ml
+   cd dask-ml
+
+You may want to add ``dask/dask-ml`` as an upstream
+
+.. code-block::none
+
+   git remote add upstream https://github.com/dask/dask-ml
+
+Creating an environment
+~~~~~~~~~~~~~~~~~~~~~~~
+
+We have an conda ``environment.yaml`` with all the dependencies. If you're using
+conda you can
+
+.. code-block:: none
+
+   conda env create -f ci/environment.yml --name=dask-ml-dev
+
+If you're using pip, check out the ``setup.py`` for the required and optional
+dependencies.
+
+Building dask-ml
+~~~~~~~~~~~~~~~~
+
+The library has some C-extensions, so installing is a bit more complicated than
+normal. If you have a compiler and everything is setup correctly, you should be
+able to install Cython and all the required dependencies.
+
+From within the repository:
+
+.. code-block:: none
+
+   python setup.py build_ext --inplace
+
+And then
+
+.. code-block:: none
+
+   python -m pip install -e .
+
+If you have any trouble with the build step, please open an issue.
+
+Conventions
```

Comment:
I'd be curious to hear other's thoughts on here.

At some point, if we add a `compute` keyword to each estimator, we'll add that here.